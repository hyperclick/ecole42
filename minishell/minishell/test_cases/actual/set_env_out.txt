Apple_PubSub_Socket_Render=/private/tmp/com.apple.launchd.y8zEsLpvYZ/Render
HOME=/Users/darugula
HOMEBREW_CACHE=/tmp/darugula/Homebrew/Caches
HOMEBREW_TEMP=/tmp/darugula/Homebrew/Temp
LC_CTYPE=UTF-8
LD_PRELOAD=
LESS=-R
LOGNAME=darugula
LSCOLORS=Gxfxcxdxbxegedabagacad
OLDPWD=/Users/darugula/git/e57/minishell/minishell
PAGER=less
PATH=/bin:/usr/bin
PWD=/Users/darugula/git/e57/minishell/minishell
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.ZRmAR1Gnac/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=388.1.3
TERM_SESSION_ID=3F749BE5-F513-4574-9C36-CC060381E6FB
TMPDIR=/var/folders/zz/zyxvpxvq6csfxvn_n00016f40009lj/T/
USER=darugula
VALGRIND_STARTUP_PWD_24133_XYZZY=/Users/darugula/git/e57/minishell/minishell
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
ZSH=/Users/darugula/.oh-my-zsh
_=/Users/darugula/.brew/bin/valgrind
__CF_USER_TEXT_ENCODING=0x2671:0x0:0x0

env finished

q = w
Apple_PubSub_Socket_Render=/private/tmp/com.apple.launchd.y8zEsLpvYZ/Render
HOME=/Users/darugula
HOMEBREW_CACHE=/tmp/darugula/Homebrew/Caches
HOMEBREW_TEMP=/tmp/darugula/Homebrew/Temp
LC_CTYPE=UTF-8
LD_PRELOAD=
LESS=-R
LOGNAME=darugula
LSCOLORS=Gxfxcxdxbxegedabagacad
OLDPWD=/Users/darugula/git/e57/minishell/minishell
PAGER=less
PATH=/bin:/usr/bin
PWD=/Users/darugula/git/e57/minishell/minishell
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.ZRmAR1Gnac/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=388.1.3
TERM_SESSION_ID=3F749BE5-F513-4574-9C36-CC060381E6FB
TMPDIR=/var/folders/zz/zyxvpxvq6csfxvn_n00016f40009lj/T/
USER=darugula
VALGRIND_STARTUP_PWD_24133_XYZZY=/Users/darugula/git/e57/minishell/minishell
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
ZSH=/Users/darugula/.oh-my-zsh
_=/Users/darugula/.brew/bin/valgrind
__CF_USER_TEXT_ENCODING=0x2671:0x0:0x0
q=w

q =
Apple_PubSub_Socket_Render=/private/tmp/com.apple.launchd.y8zEsLpvYZ/Render
HOME=/Users/darugula
HOMEBREW_CACHE=/tmp/darugula/Homebrew/Caches
HOMEBREW_TEMP=/tmp/darugula/Homebrew/Temp
LC_CTYPE=UTF-8
LD_PRELOAD=
LESS=-R
LOGNAME=darugula
LSCOLORS=Gxfxcxdxbxegedabagacad
OLDPWD=/Users/darugula/git/e57/minishell/minishell
PAGER=less
PATH=/bin:/usr/bin
PWD=/Users/darugula/git/e57/minishell/minishell
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.ZRmAR1Gnac/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=388.1.3
TERM_SESSION_ID=3F749BE5-F513-4574-9C36-CC060381E6FB
TMPDIR=/var/folders/zz/zyxvpxvq6csfxvn_n00016f40009lj/T/
USER=darugula
VALGRIND_STARTUP_PWD_24133_XYZZY=/Users/darugula/git/e57/minishell/minishell
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
ZSH=/Users/darugula/.oh-my-zsh
_=/Users/darugula/.brew/bin/valgrind
__CF_USER_TEXT_ENCODING=0x2671:0x0:0x0
