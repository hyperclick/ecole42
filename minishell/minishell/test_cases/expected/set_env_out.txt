Apple_PubSub_Socket_Render=/private/tmp/com.apple.launchd.jQT3YVVlEv/Render
HOME=/Users/darugula
HOMEBREW_CACHE=/tmp/darugula/Homebrew/Caches
HOMEBREW_TEMP=/tmp/darugula/Homebrew/Temp
LC_CTYPE=UTF-8
LD_PRELOAD=
LESS=-R
LOGNAME=darugula
LSCOLORS=Gxfxcxdxbxegedabagacad
OLDPWD=/Users/darugula/git/e58/minishell/minishell
PAGER=less
PATH=/bin:/usr/bin
PWD=/Users/darugula/git/e58/minishell/minishell
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.QwNGMF50Ec/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=388.1.3
TERM_SESSION_ID=3E527B4E-8346-4659-AD97-E4CCD022ADC1
TMPDIR=/var/folders/zz/zyxvpxvq6csfxvn_n00016f40009lj/T/
USER=darugula
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
ZSH=/Users/darugula/.oh-my-zsh
_=/Users/darugula/git/e58/minishell/minishell/./ut
__CF_USER_TEXT_ENCODING=0x2671:0x0:0x0

env finished

q = w
Apple_PubSub_Socket_Render=/private/tmp/com.apple.launchd.jQT3YVVlEv/Render
HOME=/Users/darugula
HOMEBREW_CACHE=/tmp/darugula/Homebrew/Caches
HOMEBREW_TEMP=/tmp/darugula/Homebrew/Temp
LC_CTYPE=UTF-8
LD_PRELOAD=
LESS=-R
LOGNAME=darugula
LSCOLORS=Gxfxcxdxbxegedabagacad
OLDPWD=/Users/darugula/git/e58/minishell/minishell
PAGER=less
PATH=/bin:/usr/bin
PWD=/Users/darugula/git/e58/minishell/minishell
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.QwNGMF50Ec/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=388.1.3
TERM_SESSION_ID=3E527B4E-8346-4659-AD97-E4CCD022ADC1
TMPDIR=/var/folders/zz/zyxvpxvq6csfxvn_n00016f40009lj/T/
USER=darugula
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
ZSH=/Users/darugula/.oh-my-zsh
_=/Users/darugula/git/e58/minishell/minishell/./ut
__CF_USER_TEXT_ENCODING=0x2671:0x0:0x0
q=w

q =
Apple_PubSub_Socket_Render=/private/tmp/com.apple.launchd.jQT3YVVlEv/Render
HOME=/Users/darugula
HOMEBREW_CACHE=/tmp/darugula/Homebrew/Caches
HOMEBREW_TEMP=/tmp/darugula/Homebrew/Temp
LC_CTYPE=UTF-8
LD_PRELOAD=
LESS=-R
LOGNAME=darugula
LSCOLORS=Gxfxcxdxbxegedabagacad
OLDPWD=/Users/darugula/git/e58/minishell/minishell
PAGER=less
PATH=/bin:/usr/bin
PWD=/Users/darugula/git/e58/minishell/minishell
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.QwNGMF50Ec/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=388.1.3
TERM_SESSION_ID=3E527B4E-8346-4659-AD97-E4CCD022ADC1
TMPDIR=/var/folders/zz/zyxvpxvq6csfxvn_n00016f40009lj/T/
USER=darugula
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
ZSH=/Users/darugula/.oh-my-zsh
_=/Users/darugula/git/e58/minishell/minishell/./ut
__CF_USER_TEXT_ENCODING=0x2671:0x0:0x0
